### ISSUE TITLE
Unclear documentation of Property bag entity types

### ISSUE THREAD

**Issue by rami-o-hassan** [dotnet/EntityFramework.Docs#3198]
Please amend the code example under "Property bag entity types" to include how instances of a property bag could be added to the context and how they are queried. I spent hours trying to understand how this works, but I keep getting the following error no matter what I try:

Cannot create a DbSet for 'Dictionary<string, object>' because it is configured as an shared type entity type. Access the entity type via the `Set` method overload that accepts an entity type name

I see this as a documentation shortcoming. More elaborate example is required.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 550a3c36-7f1d-efa7-3ed8-769720168da4
* Version Independent ID: 627c4301-e00d-826e-d545-3a244305dec9
* Content: [Shadow and Indexer Properties - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/shadow-properties)
* Content Source: [entity-framework/core/modeling/shadow-properties.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/modeling/shadow-properties.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @AndriySvyryd
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
@rami-o-hassan 

> Cannot create a DbSet for 'Dictionary<string, object>' because it is configured as an shared type entity type. Access the entity type via the Set method overload that accepts an entity type name

Can you provide some more details about what was not clear in this exception message?

**Comment by rami-o-hassan** [comment]
I don't think it is about the exception. I mean, thank you, of course, for trying to learn more about the problem I am facing, but this is basically more about the lack of enough details in this page of documentation. The exception was just my argument that I can't get this to work, and neither the exception message nor the documentation is helping. 

I looked more thoroughly in the issues, and I can see my issue here is a duplicate of #3031, which happens to have better description, so I guess this issue here should be closed.
